packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Fixed propType warning if selectedDate prop is an empty string.
```

Comment:
This might need rephrased. If it's a fix it should move under a fixed section. If we're considering this an enhancement we might just find an alternative to the word fixed.

Reply:
Allow empty string to represent no selected date

Updated the selectedDate to accept an empty string when no date is selected

Reply:
Changed to use `Fix` and updated the description since it used to crash if the selectedProp is not ISO compliant and unable to convert to a moment object.
https://github.com/cerner/terra-framework/pull/607/commits/fa9fadf949ff404eb5b8fb2b5a5ca40e414af04b